Documentation/building/unix-instructions.md

```diff
@@ -59,3 +59,19 @@ If you see errors along the lines of `SendFailure (Error writing headers)` you m
 ```sh
 mozroots --import --sync
 ```
+
+## Steps to Build on Ubuntu
+
+*Note: verified on Ubuntu 14.04 LTS*
+
+1. Install git `sudo apt-get install git`
+2. Clone the corefx repo `git clone https://github.com/dotnet/corefx.git`
+3. Install curl-dev `sudo apt-get install libcurl4-openssl-dev`
```

Comment:
@mellinoe it's not nit-picking, I appreciate perfection :smile: And yes, we can go with clang 3.5 instead of 3.6.  Let me go ahead and just run the prereqs install and build on a fresh machine again with clang 3.5 this time just to verify our thinking.  I should have this done shortly.